tsconfig.json

```diff
@@ -17,6 +17,7 @@
     "preserveConstEnums": true,
     "sourceMap": true,
     "strict": true,
+    "strictFunctionTypes": false,
```

Comment:
See https://github.com/DefinitelyTyped/DefinitelyTyped/issues/21206

Reply:
Should we add a TODO?

Reply:
I want to, but JSON doesn't support comments :(

Reply:
Haha, Iif I really need one, I usually add "_comment": "Set this to true once DefinitelyTyped/DefinitelyTyped#21206 is fixed." ;)

Reply:
Haha

```json
{
  "TODOS": [
    "Set this to true once DefinitelyTyped/DefinitelyTyped#21206 is fixed.",
    "Remove this key from JSON",
    "Stop using JSON"
}
```